rest/model/task.go

```diff
@@ -18,7 +18,6 @@ type APITask struct {
 	Id                 APIString        `json:"task_id"`
 	CreateTime         APITime          `json:"create_time"`
 	DispatchTime       APITime          `json:"dispatch_time"`
-	PushTime           APITime          `json:"push_time"`
```

Comment:
Should we keep this in the API model, and just copy the CreateTime value to PushTime, since they're almost the same?

Reply:
Furthermore, frontend js is using a push_time field on some objects. I can't determine what type they are though.

Reply:
this value is populated from other sources, I think, in the front end code, and is just poor naming on the front end side.

Reply:
Can you also purge push_time from the smoke test data?